tap_sftp_files/__init__.py

```diff
@@ -158,6 +158,33 @@ def download(args):
 
     if port:
         connection_config['port'] = int(port)
+    
+    # initialize sftp connection with paramiko to get the current working directory and list the contents of the root directory
+    sftp = pysftp.Connection(host, username=config['username'], password=config.get('password', config.get('private_key')), port=int(port))
```

Comment:
Initializing the connection like this without `with` means it will stay open and not be properly closed in some cases. I think you want to use the `with` syntax to avoid that

Also think if we're going to have these logs it should be behind a config flag